spec/controllers/admin/oral_history_requests_controller_spec.rb

```diff
@@ -104,56 +107,11 @@
         expect(oral_history_access_request.delivery_status).to eq "rejected"
 
         last_email = ActionMailer::Base.deliveries.last
-        expect(last_email.subject).to eq "Science History Institute: Your request"
+        expect(last_email.subject).to eq "Science History Institute Oral History Request: #{oral_history_access_request.work.title}"
         expect(last_email.from).to eq [ScihistDigicoll::Env.lookup(:oral_history_email_address)]
+        expect(last_email.body).to match /Unfortunately we could <b>not<\/b> approve your request/
         expect(last_email.body).to include(message)
       end
-
```

Comment:
It looks to me like maybe you accidentally removed some useful specs here for new (now only) func, rather than just moved them around? Can you confirm?  Or I can checkout the code and spend more time with it confirmiming myself!

I am just suspicious of how many lines are removed in the diff, without corresponding lines being added. I added lots of new tests for the new func that we DO want to preserve, although we don't need to label it as "new functionality" anymore of course!

Reply:
Will absolutely look into this - thanks for checking.

Reply:
I looked it over, it's actually all good!